htdocs/societe/project.php

```diff
@@ -126,7 +126,59 @@
 
 
 	// Projects list
-    $result = show_projects($conf, $langs, $db, $object, $_SERVER["PHP_SELF"].'?socid='.$object->id, 1, $newcardbutton);
+	$result=show_projects($conf, $langs, $db, $object, $_SERVER["PHP_SELF"].'?socid='.$object->id, 1, $addbutton);
+	print '<br><br><br>';
+	if(!empty($user->rights->projet->lire)) {
+	$draft =	"Brouillon <img src=\"/htdocs/theme/eldy/img/statut0.png\" alt=\"\" title=\"Brouillon\" class=\"inline-block\">";
```

Comment:
Hard coded label is not allowed.

Reply:
A lot of errors reported by quality tools must also be fixed. Also a screenshot to show what this feature bring is welcome.